components/HeaderNav/HeaderNav.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
+// import * as React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import CreateEventForm from '../CreateEventForm/CreateEventForm';
 import { StyleSheet, View, Text, Button } from 'react-native';
 
-export default function HeaderNav() {
+export default function HeaderNav({ navigation }) {
   return (
     <View style={styles.nav}>
       <View style={styles.navButtonContainer}>
-        <View style={styles.navButton}><Button title='change date'/></View>
-        <View style={styles.navButton}><Button title='create activity'/></View>
-        <View style={styles.navButton}><Button title='log out'/></View>
+        <View style={styles.navButton}><Button title='CHANGE DATE'/></View>
+        <View style={styles.navButton}><Button onPress={() => navigation.navigate('CreateEventForm')}  title='CREATE EVENT'/></View>
```

Comment:
Is this kind of our new style of router and redirecting the page to another page?